java/flight/src/main/java/org/apache/arrow/flight/GenericOperation.java

```diff
@@ -17,6 +17,9 @@
 
 package org.apache.arrow.flight;
 
+/**
+ * Unused?.
```

Comment:
I'm not sure what this is for and I don't see where it is used either, maybe @jacques-n can comment?